auditwheel/elfutils.py

```diff
@@ -79,7 +79,11 @@ def elf_references_PyFPE_jbuf(elf: ELFFile) -> bool:
 
 def elf_is_python_extension(fn, elf) -> Tuple[bool, Optional[int]]:
     modname = basename(fn).split('.', 1)[0]
-    module_init_f = {'init' + modname: 2, 'PyInit_' + modname: 3}
+    module_init_f = {
+        'init' + modname: 2,
+        'PyInit_' + modname: 3,
+        '_cffi_pypyinit_' + modname: 2,
```

Comment:
Those are versions of Python used to determine if extension may use ucs-2 functions. Corresponding code is in ``auditwheel.wheel_abi.get_wheel_elfdata``. In case of CPython those numbers are really Python version but in case if pypy cffi python version cannot be derived from the name of module init function so I left 2 there.